solcore/graphing/Custom_Colours.py

```diff
@@ -9,48 +9,49 @@
 import os
 import re
 from cycler import cycler
```

Comment:
```suggestion
from cycler import cycler, Cycler
```
The class is `Cycler` with capital `C`, and so it needs to be changed below.